include/bluetooth/services/ots_client.h

```diff
@@ -262,11 +276,11 @@ int bt_otc_select_prev(struct bt_conn *conn,
  *
  *  @return int         0 if success, ERRNO on failure.
  */
-int bt_otc_obj_metadata_read(struct bt_conn *conn,
-			     struct bt_otc_instance_t *otc_inst,
-			     uint8_t metadata);
+int bt_otc_read_object_metadata(struct bt_conn *conn,
```

Comment:
Given the rename to `ots_client`, so all occurences of `bt_otc` be named to `bt_ots_client`?

Reply:
Done now - at least in the API